src/models/account.es6.js

```diff
@@ -4,6 +4,16 @@ class Account extends Base {
   constructor(props) {
     props._type = 'Account';
     super(props);
+
+    var account = this;
+
+    this.validators = {
+      thingId: function() {
+        var thingId = this.get('thingId');
+
+        return Base.validators.thingId(thingId, '2');
```

Comment:
The `thingId` validator should also be optional - it should only test the regex _if_ it is not undefined, such as when creating a new Account to be `POST`ed.